app/models/calendar.rb

```diff
@@ -14,21 +14,34 @@ def self.find(slug)
     end
   end
 
-  attr_reader :slug, :title, :description
+  def self.file_name(slug)
+    slug == "gwyliau-banc" ? "bank-holidays" : slug
+  end
+
+  def years_covered
```

Comment:
I think this method is missing a test.

Reply:
Method no longer exists, so marking this as resolved.